backend/apps/admin/src/contest/contest.resolver.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  BadRequestException,
+  InternalServerErrorException,
+  NotFoundException,
+  ParseIntPipe,
+  UnprocessableEntityException
+} from '@nestjs/common'
+import { Args, Context, Mutation, Query, Resolver } from '@nestjs/graphql'
+import { AuthenticatedRequest, UseRolesGuard } from '@libs/auth'
+import {
+  ActionNotAllowedException,
+  EntityNotExistException,
+  UnprocessableDataException
+} from '@libs/exception'
+import { CursorValidationPipe } from '@libs/pipe'
+import { Problem } from '@admin/@generated'
+import { Contest } from '@admin/@generated/contest/contest.model'
+import { ContestService } from './contest.service'
+import { CreateContestInput } from './model/contest.input'
+import { UpdateContestInput } from './model/contest.input'
+import { PublicizingRequest } from './model/publicizing-request.model'
+
+@Resolver(() => Contest)
+export class ContestResolver {
+  constructor(private readonly contestService: ContestService) {}
+
+  @Query(() => [Contest])
+  async getContests(
+    @Args('take', ParseIntPipe) take: number,
+    @Args('groupId', ParseIntPipe) groupId: number,
+    @Args('cursor', { nullable: true }, CursorValidationPipe) cursor?: number
+  ) {
+    return await this.contestService.getContests(take, groupId, cursor)
+  }
+
+  @Mutation(() => Contest)
+  async createContest(
+    @Args('input') input: CreateContestInput,
+    @Args('groupId', ParseIntPipe) groupId: number,
```

Comment:
Open Space에 들어가 있을 때는 프론트엔드에서 그룹의 ID를 알지 못해서, `groupId` 값을 안 넣어줄 거예요! 그래서 아래처럼 항상 디폴트 값을 받도록 설정해주셔야 합니다ㅎㅎ

```suggestion
    @Args('groupId', { defaultValue: OPEN_SPACE_ID }, ParseIntPipe) groupId: number,
```